requirements/run.in

```diff
@@ -0,0 +1,5 @@
+aiokafka==0.12.0
+kafka-python-ng==2.2.3
+tenacity==8.2.3
```

Comment:
`tenacity` comes from `kytos` deps, so no need to add here again, `packaging` and `typing-extensions` can be removed.